ddtrace/constants.py

```diff
@@ -17,3 +17,5 @@
 
 MANUAL_DROP_KEY = 'manual.drop'
 MANUAL_KEEP_KEY = 'manual.keep'
+
+LOG_SPAN_KEY = '__datadog_log_span'
```

Comment:
copied format from https://github.com/DataDog/dd-trace-py/blob/c576eaf0ffc4d89bc997c3d907d0421c76a498b8/ddtrace/contrib/tornado/constants.py#L7. thought top level constants was a good place to put it to avoid any cyclical dependencies. if there's another constants to add to, lmk